graph/kosaraju.cpp

```diff
@@ -4,55 +4,61 @@
 
 #include <iostream>
 #include <vector>
+#include <stack>
 
-using namespace std;
 
 /**
  * Iterative function/method to print graph:
  * @param a[] : array of vectors (2D)
  * @param V : vertices
  * @return void
  **/
-void print(vector<int> a[], int V) {
+void print(const std::vector< std::vector<int> > &a, int V) {
     for (int i = 0; i < V; i++) {
-        if (!a[i].empty())
-            cout << "i=" << i << "-->";
-        for (int j = 0; j < a[i].size(); j++) cout << a[i][j] << " ";
-        if (!a[i].empty())
-            cout << endl;
+        if (!a[i].empty()) {
+            std::cout << "i=" << i << "-->";
+        }
+        for (int j : a[i]) {
+            std::cout << j << " ";
+        }
+        if (!a[i].empty()) {
+            std::cout << std::endl;
+        }
     }
 }
 
 /**
  * //Recursive function/method to push vertices into stack passed as parameter:
  * @param v : vertices
  * @param &st : stack passed by reference
- * @param vis[] : array to keep track of visited nodes (boolean type)
+ * @param &vis : array to keep track of visited nodes (boolean type)
```

Comment:
```suggestion
 * @param vis  array to keep track of visited nodes (boolean type)
```
the colons in the param descriptions can be removed